docs/documentation/workflows.md

```diff
@@ -341,39 +341,30 @@ In other words if a Kubernetes Dependent Resource depends on another dependent r
 
 ## Explicit Managed Workflow Invocation
 
-Managed workflow is execution before the `reconcile(Resource,Context)` is called. There are certain situations however
-when before the invocation there is additional validation or other logic needs to be executed or in some cases
-event the workflow execution skipped. For this reason managed workflow explicit invocation is possible.
-
-```java
-
-@Workflow(explicitInvocation = true,
-    dependents = @Dependent(type = ConfigMapDependent.class))
-@ControllerConfiguration
-public class WorkflowExplicitInvocationReconciler
-    implements Reconciler<WorkflowExplicitInvocationCustomResource> {
-    
-  @Override
-  public UpdateControl<WorkflowExplicitInvocationCustomResource> reconcile(
-      WorkflowExplicitInvocationCustomResource resource,
-      Context<WorkflowExplicitInvocationCustomResource> context) {
-    
-     // additional logic before the workflow is executed
-      
-     context.managedWorkflowAndDependentResourceContext().reconcileManagedWorkflow();
```

Comment:
@metacosm why did you delete sample code from here?

Reply:
Because it doesn't add much, imo, and would need to be kept up to date (as are the other samples)…

Reply:
will merge this now as it is, when we migrate to docsy, would be nice to revisit the principles for the dox though